tasks/main.yml

```diff
@@ -1,6 +1,23 @@
 ---
 
-- name: Updating cache
+- name: Check server uptime
+  shell: cat /proc/uptime
```

Comment:
We can use something like `cut -d. -f1 /proc/uptime` to grab the amount of seconds the system has been up. `cut` should be by default on all systems in question. Rounding the value makes it easier to deal with.